src/TagLib/Ogg/XiphComment.cs

```diff
@@ -307,10 +345,24 @@ public ByteVector Render (bool addFramingBit)
 		///    fields in the current instance.
 		/// </value>
 		public uint FieldCount {
-			get {
+			get
+			{
 				uint count = 0;
-				foreach (string [] values in field_list.Values)
+				foreach (string[] values in field_list.Values)
 					count += (uint) values.Length;
+
+				// If the pictures array is loaded and not in sync
+				// with the underlying fields, adjust the field count
+				if (pictures != null && picture_fields_dirty) {
+					foreach (string fieldName in PICTURE_FIELDS) {
+						string[] value;
```

Comment:
Since `value` is a keyword and in particular in a property it'd probably be good to avoid using it here.

Reply:
Fixed in 3d1dc1f30b97dd148f8d2f564716cb441e833cc3